deltacat/tests/compute/test_util_common.py

```diff
@@ -215,3 +219,28 @@ def offer_iso8601_timestamp_list(
             )
         )
     return res
+
+
+def assert_compaction_audit(
+    compactor_version: CompactorVersion, compaction_audit: CompactionSessionAuditInfo
+):
+    if compactor_version == CompactorVersion.V2:
```

Comment:
These are not definitely all attributes. Look at the actual audit file and ensure every one among them is there.

Reply:
These assertions are based on what `compaction_audit` updates are made in the refactored `compaction_session` and `compaction_utils`:
```
{'deltacatVersion': '1.1.11', 
'rayVersion': '2.32.0', 
'auditUrl': 's3://test-compaction-artifacts-bucket/compaction-audit/dadad0e699a014b0db94016217c90c63ebd81d80.json', 
'hashBucketCount': 3, 
'compactorVersion': 'V2', 
'inputSizeBytes': 6461, 
'inputFileCount': 4, 
'estimatedInMemorySizeBytesDuringDiscovery': 280, 
'uniformDeltasCreated': 1, 
'deltaDiscoveryTimeInSeconds': 0.0067308819852769375, 
'hashBucketTimeInSeconds': 0.01295870600733906, 
'hashBucketInvokeTimeInSeconds': 0.012178680044598877, 
'hashBucketResultSize': 464, 
'totalClusterObjectStoreMemoryBytes': 4091910144.0, 
'totalClusterMemoryBytes': 8183820288.0, 
'totalObjectStoreMemoryUsedBytes': 0.0, 
'hashBucketPostObjectStoreMemoryUsedBytes': 0.0, 
'hashBucketResultWaitTimeInSeconds': 0.001363515853881836, 
'hashBucketTaskPeakMemoryUsedBytes': 314687488.0, 
'hashBucketProcessedSizeBytes': 284, 
'inputRecords': 20, 
'mergeTimeInSeconds': 0.04835349402856082, 
'mergeInvokeTimeInSeconds': 0.04747437103651464, 
'mergeResultSize': 29610, 
'mergePostObjectStoreMemoryUsedBytes': 0.0, 
'mergeResultWaitTimeInSeconds': 1406994735.0460403, 
'mergeTaskPeakMemoryUsedBytes': 314687488.0, 
'recordsDeduped': 10, 
'recordsDeleted': 0, 
'compactionTimeInSeconds': 0.5707382939290255, 
'peakMemoryUsedBytesCompactionSessionProcess': 314687488, 
'untouchedFileCount': 0, 
'untouchedFileRatio': 0.0, 
'untouchedRecordCount': 0, 
'untouchedSizeBytes': 0, 
'outputFileCount': 3, 
'outputSizeBytes': 4760, 
'outputSizePyarrowBytes': 292, 
'peakMemoryUsedBytesPerTask': 314687488.0, 
'pyarrowVersion': '12.0.1', 
'telemetryTimeInSeconds': 0.001723743975162506}
```
This is a sample one from one of the `rebase_then_incremental` test cases. Should I test for all of these values? My  main worry is that values that are 0 or 0.0 would fail `not None` assertions.

Reply:
Yes please. 0.0 won't fail not None.